docs/topics/miscellneous_details_I_may_forget.md

```
@@ -0,0 +1,8 @@
+# MDIMF
+
+## Kotlin related
+
+- smart casting doesn't work for members, they might be null in race-conditions
+    - wrap nullable member with ?.let block
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)